crates/db_views_actor/src/community_person_ban_view.rs

```diff
@@ -12,16 +12,18 @@ impl CommunityPersonBanView {
     pool: &mut DbPool<'_>,
     from_person_id: PersonId,
     from_community_id: CommunityId,
-  ) -> Result<Self, Error> {
+  ) -> Result<bool, Error> {
     let conn = &mut get_conn(pool).await?;
-    community_person_ban::table
-      .inner_join(community::table)
-      .inner_join(person::table)
-      .select((community::all_columns, person::all_columns))
-      .filter(community_person_ban::community_id.eq(from_community_id))
-      .filter(community_person_ban::person_id.eq(from_person_id))
-      .order_by(community_person_ban::published)
-      .first::<CommunityPersonBanView>(conn)
-      .await
+    select(exists(
+      community_person_ban::table
+        .inner_join(community::table)
+        .inner_join(person::table)
+        .select((community::all_columns, person::all_columns))
```

Comment:
You should probably be able to remove this select, or if you can't, just select the `community_person_ban::id`

Reply:
true